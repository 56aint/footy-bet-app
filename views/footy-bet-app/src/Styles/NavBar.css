.navbar {
  background-color: rgb(49, 16, 197);
  overflow: hidden;
 /*  margin-left: 160px; */
  padding: 0 10px;
}
ul{
  float: left;
  display: flex;
  justify-content: center;
  align-items: center;
}

ul li{
  list-style: none;
  margin: 62px;
  margin-top: 27px;
  font-size: 20px;

}

.navbar a {
  color: white;
  font-weight: 700;
  letter-spacing: 2px;
  text-decoration: none;
  background: rgb(41, 38, 70);
  padding: 5px 0px;
  display: inline-block;
  width: auto;
  text-align: center;
  border-radius: 5px;
}

.navbar a:hover {
  color: #17bbd8;
}



.loading {
  margin: auto;
  width: auto;
  color: rgb(69, 9, 129);
  position: relative;
  text-align: center;
}
