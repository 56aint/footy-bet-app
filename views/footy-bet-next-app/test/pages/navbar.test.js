import React from 'react';
import { render, screen } from '../test-utils';
import NavBar from '../../Components/navbar';

describe('<NavBar />', () => {
  it('should render a component resembling the snapshot', () => {
    const { asFragment } = render(<NavBar />);

    expect(asFragment).toMatchSnapshot();
  });
  it('renders without crashing', () => {
    const navBarText = 'Live Football';

    render(<NavBar />);
    expect(screen.getByText(navBarText)).toBeInTheDocument();
  });
  it('should consider nav as a navigation role', () => {
    render(<NavBar />);
    const nav = screen.getByRole('navigation');

    expect(nav).toBeInTheDocument();
  });
  it('should consider ul as a list role', () => {
    render(<NavBar />);
    const list = screen.getByRole('list');

    expect(list).toBeInTheDocument();
  });

  it('presence of link tags', () => {
    render(<NavBar />);
    const link = screen.getAllByRole('link');

    expect(link).toBeTruthy();
  });
});
