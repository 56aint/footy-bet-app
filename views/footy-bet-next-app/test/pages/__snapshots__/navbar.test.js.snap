// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar /> should render a component resembling the snapshot 1`] = `[Function]`;
